packages/components/src/navigation/menu/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronLeft } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { ROOT_MENU } from '../constants';
+import { useNavigationContext } from '../context';
+import {
+	MenuBackButtonUI,
+	MenuTitleUI,
+	MenuUI,
+} from '../styles/navigation-styles';
+import { NavigationMenuContext } from './context';
+import { useNavigationTreeMenu } from './use-navigation-tree-menu';
+
+export default function NavigationMenu( props ) {
+	useNavigationTreeMenu( props );
```

Comment:
I'm aware I'm annoying but... mind moving this after the props destructuring, in order to keep all hooks together? 😄

Reply:
Done